source/features/rgh-feature-descriptions.tsx

```diff
@@ -140,12 +138,12 @@ function init(signal: AbortSignal): void {
 	observe('#repos-sticky-header', add, {signal});
 }
 
+// eslint-disable-next-line unicorn/better-regex -- No "\/"
+const featureUrlRegex = /^([/]refined-github){2}[/]blob[/][^/]+[/]source[/]features[/][^.]+[.](tsx|css)$/;
```

Comment:
Nice job

Reply:
I hate it 😂 but the alternative is 2 detections + 1 that uses `pathname.split()` and 3 `===`

Looking forward to pattern matching https://github.com/tc39/proposal-pattern-matching